setup.cfg

```diff
@@ -22,6 +21,9 @@ norecursedirs = .git testing_config
 
 [flake8]
 exclude = .venv,.git,.tox,docs,venv,bin,lib,deps,build
+# To work with Black
+max-line-length = 88
+ignore = E501, W503, E203
```

Comment:
Perhaps comment what these exclusions are?

Reply:
It comment out what is defined on ADR and needed for black. They allready documented that on here manual, do we realy need double entry?

Reply:
Done